### DISCUSSION TITLE
marking

### DISCUSSION THREAD

**Discussion by Dixxi0509** [tahoma2d/tahoma2d#1412] [Q&A]
hey question 

is there i guess like a marker to mark your frames like lets say  you had an audio in the timline and it changes at lets say at 129 and when scrubing the audio you're at around 70 frames [sorry  if it doesnt make sense]  if not i think this would be a good idea for timing!

**Comment by manongjohn** [comment]
There are 3 ways you can mark stuff...

- Cell Marks for marking specific level frames on the timeline/xsheet. Right-click any cell. Note: does not move when shifting frames.
- Navigation Tags for marking scene frames in the timeline/xsheet. Right-click the frame range area
- Memos to add Post-It like notes anywhere in the timeline/xsheet.  There is a button in the timeline/xsheet to add memos.

**Comment by artisteacher** [comment]
You can also use Note levels to make timing notations or etc.
